build.gradle

```diff
@@ -191,6 +191,9 @@ dependencies {
 
   optional("org.springframework.boot:spring-boot-configuration-processor")
 
+  compile("org.thymeleaf:thymeleaf:${thymeleafVersion}")
+  compile("org.thymeleaf:thymeleaf-spring4:${thymeleafVersion}")
```

Comment:
Use `org.springframework.boot:spring-boot-starter-thymeleaf"` instead of defining dependencies manually.
Read https://spring.io/guides/gs/serving-web-content/